docs/source/quick_start.rst

```diff
@@ -34,11 +34,25 @@ the *keyring*. Here's an example of a decrypted keyring file ::
 
     {
       "lochness": {
+        "REDCAP": {
+          "example": {
```

Comment:
These things are becoming very inconsistent! Below we have `redcap.StudyA`. Why do we have `redcap.example` here?

Reply:
Between my two comments, at least this should be `StudyA` although that would be inconsistent with other modules below. Notably, you also used the `StudyA` string in your created templates.